### PR TITLE
Add new filemanager subcommand for uploading files to the File Manager

### PR DESCRIPTION
**Pull request by gcorne**
This PR adds a new `filemanager` subcommand for uploading files to the File Manager.

Part of #13 

- [x] Update documentation

### CODE REVIEW

File: packages/cms-cli/README.md [hunk 1]
```diff
@@ -96,3 +96,11 @@ Authenticate against a portal using OAuth2
...
 hs auth oauth2
 ```
+
+Upload a file or directory to the File Manager
+
+```bash
+hs filemanager upload --portal=DEV [src] [dest]
+```
+
+**Note:** Uploading to the File Manager is only supported when using the `apiKey` at this point.
```
Comment (gcorne) [inline packages/cms-cli/README.md]: @williamspiro how do you feel about documenting the command before we have done the work needed to support OAuth.

Comment (williamspiro) [reply packages/cms-cli/README.md]: I think that is okay as it will allow us to get feedback faster  👍

**Comment by williamspiro** [comment]
This is awesome 🚀  2 things I notice
1. It might be useful in logging to distinguish a file was uploaded to the File Manager vs. the Design Manager? 
2. What are the subcommands of `filemanager <subcommand>`? I tried `watch` and `fetch` but neither work, so I am guessing only upload is support at the moment.

**Comment by gcorne** [comment]
> What are the subcommands of filemanager <subcommand>? I tried watch and fetch but neither work, so I am guessing only upload is supported at the moment.

Yeah, you're correct that `hs filemanager upload` is the only `filemanager` command implemented at this point.

**Pull request merged**